test/brick_seeker_web/features/autocomplete_test.exs

```diff
@@ -0,0 +1,15 @@
+defmodule BrickSeekerWeb.AutoCompleteTest do
```

Comment:
I got rid of the index listing of parts and adding them (not as interested in the CRUD actions as much as the autocomplete). This feature test file can probably be deleted until we care about those actions again.